src/curl.h

```diff
@@ -398,6 +399,7 @@ class S3fsCurl
         headers_t* GetResponseHeaders() { return &responseHeaders; }
         BodyData* GetBodyData() { return &bodydata; }
         BodyData* GetHeadData() { return &headdata; }
+        CURLcode* getCurlCode() { return &curlCode; }
```

Comment:
Why does this return a pointer?

Reply:
This function should return the value of CURLcode.
Please add `const` like `CURLcode getCurlCode() const {...}`.